src/Records/CNAME.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Spatie\Dns\Records;
+
+/**
+ * @method string target()
+ */
+class CNAME extends Record
```

Comment:
Even though the types seem to be uppercased in DNS, I would like to follow regular class capitalisation conventions. So let's rename this to `Cname`. Rename the others in a similar fashion.

Reply:
I totally get you. Just think that `Mx` or `Aaaa` look even worse. 🤔
I will do it if you want. Just want to say that most won't really benefit from regular class naming.
And the `A` class name also doesn't really follow normal conventions.